documents/charter.md

```diff
@@ -1,23 +1,24 @@
-Video traffic is already 70% of the overall traffic volume on the Internet and is expected to grow to 80% by 2028.
+Video traffic is 70% of the overall traffic volume on the Internet and is expected to grow to 80% by 2028.
 Across developed and emerging markets video traffic forms 50-80% of traffic volume on mobile networks.
 New formats like short form videos have seen tremendous growth in recent years.
 These growth trends are likely to increase with new populations coming online on mobile-first markets.
 
-Mobile network operators continuously invest in network resources, including deployment of new generations or new bands of spectrum.
-Since spectrum is a limited and expensive resource, operators often make use of flow-based traffic handling such as shaping of video traffic, especially when the network is highly loaded.
-Operators cannot explicitly measure the degradation that shaping causes to end user quality of experience (QoE), making this approach open loop. 
+While mobile network operators continuously invest in new network capacity, including deployment of new generations or new bands of spectrum, these efforts are not keeping pace with growth in demand. These network operators have found it faster and less expensive to invest in shaping of video traffic on a per-flow basis, which negatively affects video stream quality. Because that traffic is encrypted, network operators cannot explicitly measure the degradation that shaping causes to end user quality of experience (QoE), making this approach open loop. 
+
+This is similar to what networks have seen in the past, such as in the 2000s when peer-to-peer (P2P) traffic posed a similar challenge and network operators were rate-shaping P2P traffic. In the United States, this practice was found to be counter to Net Neutrality (NN) [https://docs.fcc.gov/public/attachments/FCC-08-183A1.pdf]. Around this time the IETF held a workshop on Peer-to-Peer (P2P) Infrastructure, [https://www.rfc-editor.org/rfc/rfc5594.html]. Since that time, in the United States, wireline network operators have been barred from performing application-specific rate shaping, while wireless network operators have been permitted to do so. There regulator in the US, the Federal Communications Commission (FCC), currently has a proceeding underway, "Safeguarding and Securing the Open Internet", that may change this differential network management regime some time in 2024. 
```

Comment:
In the EU (+UK) the position is set by National Regulators that take guidance from the Board of European Regulators,  [BEREC](https://www.berec.europa.eu/en/all-you-need-to-know-about-the-open-internet-rules-in-the-eu-0)